src/data/analyse_de.json

```diff
@@ -0,0 +1,316 @@
+            },
+            "totalKey": ["ppa_total_warnings_cumulated"]
+        }
+    ],
+    "translations":{
+
+        "analyseBoardTitleTitle":{
+            "0": "Downloads",
+            "1": [
+                "Registrierte Tests",
+                "Testergebnisse",
+                "Testergebnisse"
+            ],
+            "2": "Teilungsverhalten",
+            "3": "Warnungen"
+        },
+
+
+
+        "analyseBoardTitleAppend":{
+            "1": "(kumuliert)",
+            "2": "",
+            "3": "(nach Kalenderwoche)",
+            "4": "(7-Tage-Mittelwert)"
+        },
+
+
+        "legendLabelMean": "7-Tage-Mittelwert:",
+        "legendLabelDaily": "Tagliche Anzahl:",
+
+        "effective_date": "",
+        "infections_published_cumulated": "",
+        "infections_published_daily": "",
+        "infections_published_7days_avg": "",
+        "infections_published_7days_sum": "",
+
+        "total": "Gesamt",
+
+
+
+
+        "app_downloads_cumulated": "Gesamt",
+        "app_downloads_android_cumulated": "Android",
+        "app_downloads_ios_cumulated": "iOS",
+
+        "app_downloads_daily": "Gesamt",
+        "app_downloads_android_daily": "Android",
+        "app_downloads_ios_daily": "iOS",
+
+        "app_downloads_7days_sum": "Gesamt",
+        "app_downloads_android_7days_sum": "Android",
+        "app_downloads_ios_7days_sum": "iOS",
+
+        "app_downloads_7days_avg": "Gesamt X̅",
+        "app_downloads_android_7days_avg": "Android X̅",
+        "app_downloads_ios_7days_avg": "iOS X̅",
```

Comment:
I wish we could somehow reuse the text so that we don't have to duplicate it? Same goes for lines 188-226, 232-262 & 267-282.